tests/kafkatest/tests/core/upgrade_test.py

```diff
@@ -112,3 +114,29 @@ def test_upgrade(self, from_kafka_version, to_message_format_version, compressio
 
         self.run_produce_consume_validate(core_test_action=lambda: self.perform_upgrade(from_kafka_version,
                                                                                         to_message_format_version))
+
+    @parametrize(from_kafka_version=str(LATEST_0_9))
+    @parametrize(from_kafka_version=str(LATEST_0_10))
+    def test_upgrade_for_cluster_id(self, from_kafka_version):
```

Comment:
The reason is that it would be faster than spinning up a whole cluster just for the cluster id. Also, we would want to test more versions. I suggest adding `LATEST_0_10` to `test_upgrade`. What do you think?